src/cxristian/Matriz_Cuadrada.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+
+public class Matriz_Cuadrada {
+    private Scanner keyboard;
+    private int[][] mat;
+    int[] diagoPrincipal;
+    int[] diagoSecundaria;
+    int suma1 = 0, suma2 = 0;
+
+    public static void main(String[] ar) {
+        Matriz_Cuadrada ma=new Matriz_Cuadrada();
+        //ma.uploadNumbers();
```

Comment:
We should avoid commented code unless is strictly necessary. Is there a reason to keep this code commented? if so, let's include the reason as part of the comment, if not, let's remove it.